packages/wallet-link/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@helium/wallet-link",
+  "version": "4.4.0",
```

Comment:
Should this package not start at `1.0.0` or do we keep all these packages in sync?